### ISSUE TITLE
Can muuri be used to recreate packery's dashboard example?

### ISSUE THREAD

**Issue by syfgkjasdkn** [haltu/muuri#411]
👋

I want to create a draggable dashboard as shown on [Packery's](https://packery.metafizzy.co) example page under "Draggable dashboard". I wonder if that is possible with muuri, possibly with a custom layout algorithm?

The main difference I see is that packery allows for empty spaces horizontally and only fills items vertically.

**Comment by niklasramo** [comment]
@syfgkjasdkn Yep, it's doable. As you guessed you just need to build your own layout algorithm.
